devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -639,128 +692,177 @@ bool YarpRobotLoggerDevice::attachAll(const yarp::dev::PolyDriverList& poly)
 
     const unsigned dofs = joints.size();
     m_bufferManager.setDescriptionList(joints);
+    if(m_sendDataRT)
+    {
+        char* tmp = std::getenv("YARP_ROBOT_NAME");
```

Comment:
I think that having this environment variable set is not mandatory. What about passing this info from configuration file? At the moment the configuration files are already robot specific

Reply:
Ah ok, then I think we can ignore this comment